tests/integration.rs

```diff
@@ -191,7 +191,7 @@ async fn test_tombstone_stream(client: &Client) -> Result<(), Box<dyn Error>> {
     debug!("Tombstone stream [{}] result: {:?}", stream_id, result);
 
     let result = client
-        .read_stream(stream_id.as_str(), &Default::default(), 1)
+        .read_stream(stream_id.as_str(), &Default::default())
```

Comment:
Shouldn't this be:

`ReadStreamOptions::default().max_count(1)` 

Or are there so few events in the stream it really doesnt matter?